baselines/bigquery-storage/src/v1beta1/big_query_storage_client.ts.baseline

```diff
@@ -74,40 +74,34 @@ export class BigQueryStorageClient {
    *     your project ID will be detected automatically.
    * @param {string} [options.apiEndpoint] - The domain name of the
    *     API remote host.
+   * @param {gax.ClientConfig} [options.clientConfig] - client configuration override.
+   *     Client configuration set in the JSON config
+   *     (`src/v1beta1/big_query_storage_client_config.json`)
+   *     can be overridden by passing `clientConfig` to the client constructor.
+   *     The detailed structure of the clientConfig can be found
```

Comment:
Why don't we just pull this detailed configuration into the README, if we think it's important to call out, that way the link won't change.

Reply:
I have removed all this docs stuff from this PR. As discussed, I'll prepare a separate markdown file in google-gax and will update all docs in a separate generator PR (today or next week).